lib/ui/huge_listview/lazy_loading_gallery.dart

```diff
@@ -132,6 +149,8 @@ class _LazyLoadingGalleryState extends State<LazyLoadingGallery> {
   void dispose() {
     _reloadEventSubscription.cancel();
     _currentIndexSubscription.cancel();
+    widget.selectedFiles.removeListener(() {});
```

Comment:
Will this actually remove the listener we just added in line 68?
Or we need to add a method and do this?

> widget.selectedFiles.addListener(methodName)

> widget.selectedFiles.removeListner(methodName)

Reply:
Just checked, it isn't getting removed.